main/main.go

```diff
@@ -164,6 +166,17 @@ func main() {
 		return
 	}
 
+	manualWarpMessages := make(map[ids.ID][]*vmtypes.WarpLogInfo)
+	for _, msg := range cfg.ManualWarpMessages {
+		sourceBlockchainID := msg.GetSourceBlockchainID()
+
+		warpMessageInfo := vmtypes.WarpLogInfo{
```

Comment:
was there any specific reason we didn't directly save the parsed `warp.UnsignedMessage` in `WarpLogInfo`? I think we should do so for manual and regular warp info, the former when we set up config, and latter when we listen for new warp log events.

Reply:
We do save the parsed Warp message in the `WarpMessageInfo` type, but it's worth noting that https://github.com/ava-labs/awm-relayer/pull/156, which has this branch as its target, removes `WarpMessageInfo` altogether and does the parsing internally in `MessageManager`.

Reply:
I mean instead of saving the message's bytes `[]byte` we can directly save the `warp.UnsignedMessage`, and not need to do multiple parses

Reply:
Yup, in the PR I linked above most of the interfaces are changed to accept a `*warp.UnsignedMessage` directly. We only need to parse once in the message manager.